lebedigital/validation.py

```diff
@@ -0,0 +1,79 @@
+from pyshacl import validate
+from rdflib import Graph, URIRef, Namespace
+from rdflib.util import guess_format
+from rdflib.namespace import SH, RDF
+
+"""
+baseDir0 = Path(__file__).resolve().parents[0]
+baseDir1 = Path(__file__).resolve().parents[1]
+baseDir2 = Path(__file__).resolve().parents[2]
+ontologyPath = os.path.join(baseDir2,'ConcreteOntology')
+metadataPath = os.path.join(baseDir0,'E-modul-processed-data/emodul_metadata.csv')
+graphPath = os.path.join(baseDir0,'E-modul-processed-data/EM_Graph.ttl')
+processedDataPath = os.path.join(baseDir0,'E-modul-processed-data')
+"""
+
+SCHEMA = Namespace('http://schema.org/')
+
+"""
+Given a path to a shacl shape and a path to an rdf file, this function tests the rdf data against the specified shacl shapes.
+The result is an rdflib graph containing the validation report, if it is empty the validation was successful.
+"""
+def test_graph(rdf_graph: Graph, shapes_graph: Graph) -> Graph:
+
+    conforms, result_graph, _ = validate(
```

Comment:
could you document the input parameters, see e.g. https://github.com/BAMresearch/probeye/blob/main/probeye/postprocessing/sampling_plots.py

Reply:
I've added the comments